### PR TITLE
Added tests for meta data fetchers

### PR DESCRIPTION
**Pull request by ahmetmircik**

### CODE REVIEW

File: hazelcast/src/test/java/com/hazelcast/internal/nearcache/impl/invalidation/RepairingTaskTest.java [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+import static com.hazelcast.internal.nearcache.impl.invalidation.RepairingTask.MAX_TOLERATED_MISS_COUNT;
+import static com.hazelcast.internal.nearcache.impl.invalidation.RepairingTask.MIN_RECONCILIATION_INTERVAL_SECONDS;
+import static com.hazelcast.internal.nearcache.impl.invalidation.RepairingTask.RECONCILIATION_INTERVAL_SECONDS;
+import static com.hazelcast.util.RandomPicker.getInt;
+import static java.util.concurrent.TimeUnit.NANOSECONDS;
+import static org.junit.Assert.assertEquals;
+import static org.mockito.Mockito.mock;
+
+@RunWith(HazelcastParallelClassRunner.class)
+@Category({Quick.class, ParallelTest.class})
```
Comment (Donnerbart) [inline hazelcast/src/test/java/com/hazelcast/internal/nearcache/impl/invalidation/RepairingTaskTest.java]: `Quick` should be `QuickTest` (reason for the compile error)

File: hazelcast/src/test/java/com/hazelcast/internal/nearcache/impl/invalidation/RepairingTaskTest.java [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+import com.hazelcast.config.Config;
+import com.hazelcast.logging.ILogger;
+import com.hazelcast.logging.Logger;
+import com.hazelcast.spi.ExecutionService;
+import com.hazelcast.spi.properties.HazelcastProperties;
+import com.hazelcast.test.HazelcastParallelClassRunner;
+import com.hazelcast.test.HazelcastTestSupport;
+import com.hazelcast.test.annotation.ParallelTest;
+import com.hazelcast.util.UuidUtil;
+import com.sun.xml.internal.bind.v2.model.annotation.Quick;
```
Comment (Donnerbart) [inline hazelcast/src/test/java/com/hazelcast/internal/nearcache/impl/invalidation/RepairingTaskTest.java]: Ah, this is the import why it doesn't fail locally in your IDE :) In IDEA you can exclude `com.sun.xml` from the auto-imports, so this won't show up again in the import options.

Comment (ahmetmircik) [inline hazelcast/src/test/java/com/hazelcast/internal/nearcache/impl/invalidation/RepairingTaskTest.java]: fixed.

File: hazelcast/src/test/java/com/hazelcast/internal/nearcache/impl/invalidation/RepairingTaskTest.java (hunk 1 above)
Comment (ahmetmircik) [inline hazelcast/src/test/java/com/hazelcast/internal/nearcache/impl/invalidation/RepairingTaskTest.java]: fixed.

**Comment by ahmetmircik** [comment]
only adding tests

**Pull request merged**